src/analyserexternalvariable.cpp

```diff
@@ -28,23 +28,23 @@ AnalyserExternalVariable::AnalyserExternalVariableImpl::AnalyserExternalVariable
 {
 }
 
-std::vector<VariableWeakPtr>::iterator AnalyserExternalVariable::AnalyserExternalVariableImpl::findDependency(const ModelPtr &model,
-                                                                                                              const std::string &componentName,
-                                                                                                              const std::string &variableName)
+std::vector<VariablePtr>::iterator AnalyserExternalVariable::AnalyserExternalVariableImpl::findDependency(const ModelPtr &model,
+                                                                                                          const std::string &componentName,
+                                                                                                          const std::string &variableName)
 {
-    return std::find_if(mDependencies.begin(), mDependencies.end(), [=](const VariableWeakPtr &v) {
-        auto variable = v.lock();
+    return std::find_if(mDependencies.begin(), mDependencies.end(), [=](const auto &v) {
+        auto variable = v;
```

Comment:
Not quite sure what this statement is doing now?

Reply:
`variable = v`? Clearly nothing. Will be removed. :)

Reply:
087f9c83